dev/cognitiveservices/data-plane/Language/analyzetext.json

```diff
@@ -2572,24 +2840,371 @@
+      "properties": {
+        "unit": {
+          "$ref": "#/definitions/SpeedUnit"
+        }
+      },
+      "required": [
+        "unit"
+      ]
+    },
+    "AreaMd": {
+      "description": "Represents the area entity Metadata model.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/BaseMd"
+        },
+        {
+          "$ref": "#/definitions/QuantityMd"
+        }
+      ],
+      "type": "object",
+      "x-ms-discriminator-value": "AreaMd",
+      "properties": {
+        "unit": {
+          "$ref": "#/definitions/AreaUnit"
+        }
+      },
+      "required": [
+        "unit"
+      ]
+    },
+    "LengthMd": {
+      "description": "Represents the length entity Metadata model.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/BaseMd"
+        },
+        {
+          "$ref": "#/definitions/QuantityMd"
+        }
+      ],
+      "type": "object",
+      "x-ms-discriminator-value": "LengthMd",
+      "properties": {
+        "unit": {
+          "$ref": "#/definitions/LengthUnit"
+        }
+      },
+      "required": [
+        "unit"
+      ]
+    },
+    "InformationMd": {
+      "description": "Represents the information (data) entity Metadata model.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/BaseMd"
+        },
+        {
+          "$ref": "#/definitions/QuantityMd"
+        }
+      ],
+      "type": "object",
+      "x-ms-discriminator-value": "InformationMd",
+      "properties": {
+        "unit": {
+          "$ref": "#/definitions/InformationUnit"
+        }
+      },
+      "required": [
+        "unit"
+      ]
+    },
+    "TemperatureMd": {
+      "description": "Represents the temperature entity Metadata model.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/BaseMd"
+        },
+        {
+          "$ref": "#/definitions/QuantityMd"
+        }
+      ],
+      "type": "object",
+      "x-ms-discriminator-value": "TemperatureMd",
+      "properties": {
+        "unit": {
+          "$ref": "#/definitions/TemperatureUnit"
+        }
+      },
+      "required": [
+        "unit"
+      ]
+    },
+    "WeightMd": {
+      "description": "Represents the weight entity Metadata model.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/BaseMd"
+        },
+        {
+          "$ref": "#/definitions/QuantityMd"
+        }
+      ],
+      "type": "object",
+      "x-ms-discriminator-value": "WeightMd",
+      "properties": {
+        "unit": {
+          "$ref": "#/definitions/WeightUnit"
+        }
+      },
+      "required": [
+        "unit"
+      ]
+    },
+    "CurrencyMd": {
+      "description": "Represents the currency entity Metadata model.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/BaseMd"
+        },
+        {
+          "$ref": "#/definitions/QuantityMd"
+        }
+      ],
+      "type": "object",
+      "x-ms-discriminator-value": "CurrencyMd",
+      "properties": {
+        "ISO4217": {
```

Comment:
Nit: should be camelCase i.e., "iso4217".

Reply:
we changed from iso4217 to ISO4217 to be more consistent across APIs

Reply:
The intention was to match the `CurrencyResolution` type in CLU (analyzeconversations.json), which had already GA'd. In fact, `CurrencyResolution` was originally in common.json. But since the approach that Text Analytics is now taking is moving away from resolutions and towards this new `CurrencyMetadata` type and other metadata types, I wonder if it's appropriate to fix the naming here now.

Reply:
It looks like this has been addressed, so I will mark it as resolved.